src/cf.data.pre

```diff
@@ -6481,6 +6482,11 @@ DOC_START
 		the HTTP standard. Enabling this feature could make you
 		liable for problems which it causes.
 
+		ignore-must-revalidate ignores any ``Cache-Control: must-revalidate``
+		headers received from a server. Doing this VIOLATES
+		the HTTP standard. Enabling this feature could make you
+		liable for problems which it causes.
+
```

Comment:
_After_ addressing other change requests, please explicitly document whether the option applies to `Cache-Control: proxy-revalidate` responses.

Reply:
Better yet, please restrict the options behaviour to only ignoring ```must-revalidate```. Leaving other controls being obeyed correctly until we have an explicit case appear needing them to be violated.

Reply:
I support Amos suggestion about restricting this option to `must-revalidate` (_if_ that can be correctly implemented without heroic efforts!), but I think this particular change request does not depend on that decision -- we should explicitly document whether this option applies to `proxy-revalidate` regardless of whether it applies, especially because it _did_ apply before it was removed in 064679e.